packages/atlas-cloudflare-worker/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@joystream/atlas-cloudflare-worker",
+  "license": "GPL-3.0",
+  "version": "1.0.0",
+  "description": "Atlas Cloudflare worker",
+  "main": "dist/worker.js",
+  "scripts": {
+    "build": "webpack",
+    "format": "prettier --write  '*.{json,js}' 'src/**/*.{js,ts}' 'test/**/*.{js,ts}'",
+    "lint": "eslint --max-warnings=0 src && prettier --check '*.{json,js}' 'src/**/*.{js,ts}' 'test/**/*.{js,ts}'"
+  },
+  "devDependencies": {
+    "@cloudflare/workers-types": "^3.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.10.0",
+    "@typescript-eslint/parser": "^5.10.0",
+    "eslint": "^8.7.0",
+    "eslint-config-prettier": "^8.3.0",
+    "prettier": "^2.5.1",
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.5.4",
```

Comment:
`typescript` `prettier` end `eslint` are installed globally, do we need it here?

Reply:
Removed